src/avram/queryable.cr

```diff
@@ -55,6 +55,15 @@ module Avram::Queryable(T)
     self
   end
 
+  # returns the number of records removed as `Int64`.
```

Comment:
Could you make the first line a summary with a new line after. “This will delete the records using the queries where clauses, or all records if no where’s are added”

Or something like that

I like that it says what it returns but maybe that can go after the part that explains what it does. So tit reads in the order that it executed